### PR TITLE
Closes BG-1247: Use final usd in nonprofit donation records

### PR DESCRIPTION
**Pull request by ap-justin**
## Explanation of the solution
* use new `finalAmountUsd` 
* move row to own component
* update csv values

image

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/pages/Admin/Charity/Donations/Row.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    </Cells>
+  );
+}
+
+const amount = (splitLiqPct: number, amount = 0) => {
+  const split = splitLiqPct / 100;
+  const amounts = {
+    total: amount,
+    liq: amount * split,
+    locked: amount * (1 - split),
```
Comment (SovereignAndrey) [inline src/pages/Admin/Charity/Donations/Row.tsx:63]:
```suggestion
    locked: amount -  liqSplitAmount,
```

File: src/pages/Admin/Charity/Donations/Row.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            size={20}
+          />
+        )}
+      </td>
+    </Cells>
+  );
+}
+
+const amount = (splitLiqPct: number, amount = 0) => {
+  const split = splitLiqPct / 100;
```
Comment (SovereignAndrey) [inline src/pages/Admin/Charity/Donations/Row.tsx:59]:
```suggestion
  const liqSplitAmount = amount * (splitLiqPct / 100);
```

File: src/pages/Admin/Charity/Donations/Row.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  const amounts = {
+    total: amount,
+    liq: amount * split,
```
Comment (SovereignAndrey) [inline src/pages/Admin/Charity/Donations/Row.tsx:62]:
```suggestion
    liq: liqSplitAmount,
```

**Review by SovereignAndrey — CHANGES_REQUESTED** [review]
@ap-justin Seems like we have a rounding or precision handling error somewhere. Almost all of the numbers are off by a penny or two (+/- $0.01 - 0.02) when I combine the two split values to get back at the total. :thinking: 

More example records from Demo Nonprofit no. 8:
image

I've left a few suggestions as a start that might improve the reliability of the numbers, if the issue is not in the `humanize()` part itself.

**Comment by ap-justin** [comment]
> @ap-justin Seems like we have a rounding or precision handling error somewhere. Almost all of the numbers are off by a penny or two (+/- $0.01 - 0.02) when I combine the two split values to get back at the total. 🤔
> 
> More example records from Demo Nonprofit no. 8: image
> 
> I've left a few suggestions as a start that might improve the reliability of the numbers, if the issue is not in the `humanize()` part itself.
> 
> UPDATE: or... if we used 4 decimals of precision for the amounts, that might give us the extras needed to get back at the totals. Will test on a few examples and circle back.

I tried rounding `liq` before deriving locked from it, and gave consistent result

image

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**